deploy/manifests/controller/helm/retina/templates/daemonset.yaml

```diff
@@ -60,7 +60,8 @@ spec:
           {{- end}}
           {{- end}}
           ports:
-          - containerPort: {{ .Values.retinaPort }}
+          - name: http
```

Comment:
can we make this name retina ? so it is specific to this daemonset,

Reply:
This port is already scoped to the daemonset pods, I think this change is not necessary. Do you absolutely want me to change this?

Reply:
I made the change.